modules/pool/invest/lib/useInvest.ts

```diff
@@ -31,6 +31,12 @@ export function useInvest() {
         amount: getUserBalanceForToken(token.address),
     }));
 
+    const hasValidUserInput =
+        !selectedInvestTokensWithAmounts.every((token) => parseFloat(token.amount) === 0) &&
+        selectedInvestTokensWithAmounts.every(
+            (token) => parseFloat(token.amount) <= parseFloat(getUserBalanceForToken(token.address)),
```

Comment:
you can't do parse float on precise comparisons like this, you'll end up with rounding errors and undesired behavior at the edges